jest.config.js

```diff
@@ -3,7 +3,8 @@ module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'jsdom',
   moduleNameMapper: {
-    '^api': '<rootDir>/api/index',
+    '^apiFunctions': '<rootDir>/apiFunctions/index',
+    '^database': '<rootDir>/database/index',
```

Comment:
Had to update this for some tests to find the right modules and run properly.

Might simplify our importing of local modules in another PR to avoid needing to use `moduleNameMapper` at all,

Reply:
we ship a huge amount of js, I think around like 400kb on most pages, we should probably do a lot more culling besides this after we finish with everything

Reply:
I _think_ changing our import pattern would reduce the module size imported onto each page so this would help across the project 🤞🏼